apps/opencode/package.json

```
@@ -39,34 +31,23 @@
     "eslint": "catalog:lint",
-    "fs-fixture": "catalog:testing",
-    "gunshi": "catalog:runtime",
-    "tinyglobby": "catalog:runtime",
+    "nano-spawn": "catalog:runtime",
     "tsdown": "catalog:",
-    "valibot": "catalog:runtime",
+    "unplugin-macros": "catalog:build",
+    "unplugin-unused": "catalog:build",
     "vitest": "catalog:testing"
   },
   "engines": {
```

ESLint

[error] Expected object keys to be in specified order. 'engines' should be before 'scripts'.

(jsonc/sort-keys)